EIPS/eip-FOCIL.md

```diff
@@ -33,68 +33,67 @@ FOCIL is a simple committee-based design improving upon previous IL mechanisms o
 
 #### CL P2P Validation Rules
 
-1. The number of transactions in the local IL does not exceed the maximum gas limit allowed.
-2. The slot of the local IL matches the current slot. Local ILs not matching the current slot should be ignored.
-3. The parent hash of the IL is recognized.
-4. The local IL is received before the local IL freeze deadline (e.g., `t=9s`) into the slot.
-5. Received two or fewer local ILs from this IL committee member (see IL equivocation section below).
-6. The local IL is correctly signed by the validator.
-7. The validator is part of the IL committee.
-8. The size of a local IL does not exceed the maximum size allowed (e.g., 8 KB).
+When nodes receive ILs from the P2P network, they perform a series of validation checks before forwarding or caching them. These rules protect against Denial-of-Service (DoS) attacks by (1) limiting ILs' byte size and (2) restricting IL proposals to a small committee of IL committee members, thereby tightly bounding the main resource consumed by the propagation of ILs. Consumption of other relevant resources, such as verification time, is minimal because the only nontrivial check performed on IL propagation is signature verification. At this stage, there is no EL verification of the transactions within ILs. This means that ILs are allowed to contain any transactions—valid or invalid—since nodes do not perform EL-side checks. This design choice is intended to avoid additional computational overhead.
+
+1. The slot of the IL matches the current slot. ILs not matching the current slot should be ignored.
+2. The parent hash of the IL is recognized.
+3. Received two or fewer ILs from this IL committee member (see IL equivocation section below).
+4. The IL is correctly signed by the validator.
+5. The validator is part of the IL committee.
+6. The size of a IL does not exceed the maximum size allowed (e.g., 8 KB).
 
 ## Specification
 
 ### Execution Layer
 
-On the execution layer, the block validity conditions are extended such that, after all of the transactions in the block have been executed, we attempt to execute each valid transaction from local ILs that was not present in the block.
+On the execution layer, the block validity conditions are extended such that, after all of the transactions in the block have been executed, we attempt to execute each valid transaction from ILs that was not present in the block.
 If one of those transactions executes successfully, then the block is invalid.
 
 Let `B` denote the current block.
 Let `S` denote the execution state following the execution of the last transaction in `B`.
 
-For each transaction `T` in local ILs, perform the following:
+For each transaction `T` in ILs, perform the following:
```

Comment:
Unrelated to this PR, but I think somewhere along these checks, we want to check if `B` is full, if it's full, then we are done